Makefile

```diff
@@ -41,8 +40,13 @@ vet:
 check: vet
 	golangci-lint run
 
+cross-build-test:
+	GOOS=linux GOARCH=amd64 go build -v -o /dev/null ./cmd/containerd-nydus-grpc
+	GOOS=windows GOARCH=amd64 go build -v -o /dev/null ./cmd/containerd-nydus-grpc
+	GOOS=darwin GOARCH=amd64 go build -v -o /dev/null ./cmd/containerd-nydus-grpc
```

Comment:
How about using github action to do a matrix build for GOOS{linux, windows, darwin} and GOARCH{amd64, arm64}?

Reply:
Thanks for the comment, fixed.